js/app.js

```diff
@@ -0,0 +1,180 @@
+// 1
+function setAlarm(employed, vacation) {
+    return employed && !vacation;
+}
+
+// 2
+function closeCompare(a, b, margin = 0) {
+    if (b - margin <= a && a <= b + margin) {
+        return 0;
+    }
+    if (a < b) {
+        return -1;
+    }
+    return 1;
+
+// 3
+    function or(a, b) {
+        return !(!a && !b);
+    }
+// true, false = true how the function works because a! is falsy and b! this would give false for !a && !b but we use !(...) so the outcome of what is inside the round brackets we invert it false is turned into truthy
+// true, true = true
+// false, true = true
+// false, false = false
+
+    function xor(a, b) {
+        return !(!a && !b ) && !(a && b); // we check of at least one of the arguments is truthy and after that && we want to ensure that both a and be are not truthy at the same time
+    }
+
+// 4
+    function _if(bool, func1, func2) {
+        if (bool) {
+            return func1();
+        }
+        return func2();
+    }
+
+// 5
+    function rentalCarCost(rentedDays) {
+        if (rentedDays >= 7) {
+            return (rentedDays * 40) - 50;
+        } else if (rentedDays >= 3) {
+            return (rentedDays * 40) - 20;
+        }
+        return rentedDays * 40;
+    }
+}
+
+// 6
+function getRealFloor(n) {
+    if (n < 0) {
+        return n;
+    }
+    else if (n === 0 || n === 1) {
+        return n;
+    }
+    else if (n < 13) {
+        return n - 1;
+    }
+    return n - 2;
+}
+
+// 7
+function rpsls(pl1,pl2) {
```

Comment:
Please don't use such names. I know it came from codewars but I already pointed out that we should use our own descriptibve names in the code

Reply:
Correct! I coded this one a time ago, before you, gave me feedback about it. So last time I was fixing the code, I didn't notice this one. 😅